addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -187,4 +187,20 @@ weSnippetEditor.Class.include({
         });
     },
 });
+
+weSnippetEditor.Editor.include({
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _isEmptyAndRemovable($el, editor) {
+        editor = editor || $el.data('snippet-editor');
+        return this._super(...arguments) || ($el.children('.s_parallax_bg').length === $el.children().length
+            && $el.text().trim() === ''
+            && !$el.hasClass('oe_structure') && (!editor || editor.isTargetParentEditable));
```

Comment:
Should be common code between web_editor and website. Especially true with the other comment: we should handle shape and video the same way (and shapes are in web_editor)